### PR TITLE
Added registry and portal security and log info

### PR DESCRIPTION
**Pull request by stuclem**
- Fixes https://github.com/vmware/vic-product/issues/318
- Fixes https://github.com/vmware/vic-product/issues/112
- Towards https://github.com/vmware/vic-product/issues/424

@sergiosagu @jzt and @reasonerjt can you please review? Thanks!

### CODE REVIEW

File: docs/user_doc/vic_vsphere_admin/appliance_logs.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+**Procedure**
+
+1. Use SSH to connect to the appliance as root user.<pre>$ ssh root@new_vic_appliance_address</pre>When prompted for the password, enter the appliance password that you specified when you deployed the appliance.
+2. To access the logs for the appliance, navigate to `/var/log`.<pre>$ cd /var/log</pre>The `/var/log` folder contains the appliance installation log, `installation.log`.
+3. To access logs for vSphere Integrated Containers Registry, navigate to `/var/log/harbor`.<pre>$ cd /var/log/harbor</pre>The `/var/log` folder contains the log files for the following services:
+
+   - `adminserver.log`: Registry administration service.
+   - `clair-db.log`: Clair database used for vulnerability scanning of images
+   - `clair.log`: Clair service used for vulnerability scanning of images
+   - `jobservice.log`
```
Comment (reasonerjt) [inline docs/user_doc/vic_vsphere_admin/appliance_logs.md]: Registry job service log

File: docs/user_doc/vic_vsphere_admin/security_reference.md [hunk 2]
```diff
@@ -10,54 +10,86 @@ The Security Reference provides information to allow you to secure your vSphere
...
-- 2376 for TLS secured port for Docker API access
-- 22 SSH when enabled with `vic-machine debug`
-- 2378 VIC admin server health and log access (HTTPS)
-- 6060 pprof debug data when enabled with --debug levels
+The vSphere Integrated Containers appliance makes the core vSphere Integrated Containers services available.
 
-Management interface:
+|Port|Protocol|Description|
+|---|---|---|
+|443|HTTPS|Connections to vSphere Integrated Containers Registry from vSphere Integrated Containers Management Portal, VCHs, and Docker clients|
```
Comment (reasonerjt) [inline docs/user_doc/vic_vsphere_admin/security_reference.md]: Please also add 4443 |HTTPS| Connection to the "notary service"

File: docs/user_doc/vic_vsphere_admin/appliance_logs.md (hunk 1 above)
Comment (stuclem) [reply docs/user_doc/vic_vsphere_admin/appliance_logs.md]: Fixed in https://github.com/vmware/vic-product/pull/979/commits/da882e288518f482a4175d6e082fba5d23fe4cba

File: docs/user_doc/vic_vsphere_admin/security_reference.md (hunk 2 above)
Comment (stuclem) [reply docs/user_doc/vic_vsphere_admin/security_reference.md]: Fixed in https://github.com/vmware/vic-product/pull/979/commits/da882e288518f482a4175d6e082fba5d23fe4cba

File: docs/user_doc/vic_vsphere_admin/appliance_logs.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+   - `mysql.log`: Embedded registry database
+   - `notary-db.log`: Notary database by Docker Content Trust
+   - `notary-server.log`: Notary server used by Docker Content Trust
+   - `notary-signer.log`: Notary image signing service used by Docker Content Trust
+   - `registry.log`: Registry service logs
+   - `ui.log`: User interface logs
+
+1. To access logs for vSphere Integrated Containers Management Portal, run the `docker logs` command.
+
+    vSphere Integrated Containers Management Portal runs as a Docker container in the appliance.<pre>$ docker logs vic-admiral</pre>
```
Comment (sergiosagu) [inline docs/user_doc/vic_vsphere_admin/appliance_logs.md]: This is correct at the moment. But we have a task in our queue (Jira VBV-1711) to move the Admiral container logs from the console to a file, and then they will be accessible in a similar way like Harbor logs are accessible. I'll let you know when that task is completed.

Comment (stuclem) [reply docs/user_doc/vic_vsphere_admin/appliance_logs.md]: Thanks @sergiosagu. I have a doc issue open to track that for 1.3, but this PR is going into a 1.2.x update.

**Pull request merged**